lib/core.js

```diff
@@ -1,19 +1,19 @@
-'use strict';
-
+'use strict'
+/*global Symbol:true */
 var asap = require('asap')
 
 // Use Symbol if it's available, but otherwise just
 // generate a random string as this is probably going
 // to be unique
 var handleSymbol = typeof Symbol === 'function' ?
-    Symbol() :
-    (
-      '_promise_internal_key_handle_' +
-      Math.random().toString(35).substr(2, 10) + '_'
-    )
+  Symbol() :
+  (
+  '_promise_internal_key_handle_' +
+  Math.random().toString(35).substr(2, 10) + '_'
+  )
```

Comment:
This is easier to read with the additional indentation for the bracketed expression.  Readability > Lintability